static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -64,6 +64,10 @@ define([
         });
     }
 
+    function outsideMobileAdsTest() {
```

Comment:
Would this be easier to read if it were `inMobileAdsTest()` 
and `if (config.switches.viewability && !inMobileAdsTest())` below?